apps/vs-code-designer/src/main.ts

```diff
@@ -29,6 +31,14 @@ const perfStats = {
 };
 
 export async function activate(context: vscode.ExtensionContext) {
+  //TODO: double check below commands are necessary; poured over from data mapper extension
+  vscode.commands.executeCommand('setContext', 'azureDataMapper.supportedDataMapDefinitionFileExts', supportedDataMapDefinitionFileExts);
```

Comment:
NIT: Problably the `azureDataMapper` its going to change `azureLogicAppsStandard`. Also lets store those commands strings in the constants file